### DISCUSSION TITLE
Ability to view claim date

### DISCUSSION THREAD

**Discussion by CodfishBender** [GriefPrevention/GriefPrevention#1196] [Ideas]
Hi, it's becoming an issue on my server where players tend to claim on top of one another. We have a server rule that a claim within 50 blocks of another may be considered griefing if the original claimant does not permit it. 

I humbly request a feature to see when claims were created. It will certainly help for any new claim disputes in the future.

**Comment by RoboMWM** [comment]
Currently not possible, except maybe creation date on the claim file. v19 is around the time that I plan to implement changes to the db (I want to change claim ids to use a timestamp).
